src/vs/editor/contrib/folding/browser/foldingModel.ts

```diff
@@ -236,6 +236,10 @@ export class FoldingModel {
 		this._decorationProvider.removeDecorations(this._editorDecorationIds);
 	}
 
+	getRegions() {
+		return this._regions;
+	}
```

Comment:
There is already a getter for this: https://github.com/microsoft/vscode/blob/89c30e1b86f941db095d9f52b128287e5039e004/src/vs/editor/contrib/folding/browser/foldingModel.ts#L40

Reply:
Right, just noticed, thanks for pointing it out